javascript/javascript3/week2/promise-visual/main.js

```diff
@@ -0,0 +1,25 @@
+const redBox = document.querySelector('ul.marks li:nth-child(1)');
+const blueBox = document.querySelector('ul.marks li:nth-child(2)');
+const greenBox = document.querySelector('ul.marks li:nth-child(3)');
+const boxes = [redBox, blueBox, greenBox];
+
+async function translateOneByOne(){
```

Comment:
May be you have not noticed, elements are not reaching at target positions.

Reply:
I did notice before- but I assumed that they are almost reaching the target. Thank you for pointing it out and now I have made the changes accordingly.